src/printer.js

```diff
@@ -2053,6 +2053,20 @@ function printJSXElement(path, options, print) {
         } else {
           children.push(print(childPath));
         }
+
+        if (
+          child.closingElement &&
+            child.closingElement.type === 'JSXClosingElement'
```

Comment:
I believe this skips cases like this:

```jsx
<div really="long" attrs="that break the line" /><sibling>hi</sibling>
```
which should break into:
```jsx
<div 
  really="long" 
  attrs="that break the line" 
/>
<sibling>hi</sibling>
```
but I think with this code would break into:
```jsx
<div 
  really="long" 
  attrs="that break the line" 
/><sibling>hi</sibling>
```